enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -43,11 +43,11 @@ import (
 )
 
 var (
-	rootDirectory          = flag.String("cache.pebble.root_directory", "", "The root directory to store the database in.")
-	blockCacheSizeBytes    = flag.Int64("cache.pebble.block_cache_size_bytes", 1000*megabyte, "How much ram to give the block cache")
-	maxInlineFileSizeBytes = flag.Int64("cache.pebble.max_inline_file_size_bytes", 1024, "Files smaller than this may be inlined directly into pebble")
-	partitions             = flagutil.New("cache.pebble.partitions", []disk.Partition{}, "")
-	partitionMappings      = flagutil.New("cache.pebble.partition_mappings", []disk.PartitionMapping{}, "")
+	rootDirectoryFlag          = flag.String("cache.pebble.root_directory", "", "The root directory to store the database in.")
+	blockCacheSizeBytesFlag    = flag.Int64("cache.pebble.block_cache_size_bytes", defaultBlockCacheSizeBytes, "How much ram to give the block cache")
+	maxInlineFileSizeBytesFlag = flag.Int64("cache.pebble.max_inline_file_size_bytes", defaultMaxInlineFileSizeBytes, "Files smaller than this may be inlined directly into pebble")
+	partitionsFlag             = flagutil.New("cache.pebble.partitionsFlag", []disk.Partition{}, "")
```

Comment:
Was this flag name change accidental?

Reply:
I renamed the flags so that people don't accidentally reference the flags in the code. Before, we were using the flags to create an Options struct, but we were also just using the flags throughout the code, instead of referencing the fields from the Options struct

If we have two of the same cache type in a migration, we need to make sure we're using the options for the current cache

Reply:
Ohhh 🤦‍♀️ I know what you're saying actually. Yes that was an accident. Will fix